packages/reason-css-parser/test/Types_test.re

```diff
@@ -0,0 +1,81 @@
+open Setup;
+open Ast_helper;
+let loc = Location.none;
+
+let compare = (input, expected, {expect, _}) => {
+  let typ = Str.type_(~loc, Recursive, [input]);
```

Comment:
Adding the Str.type_ on the test seem wrong. This should be part of the code-gen, right?

Reply:
`EmitType.gen_types` turn all `type_declarations` into `Str.type_`, i made `EmitType.gen_type` to make unit test easier